.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx lint-staged
+npx lint-staged
```

Comment:
Can you merge the latest from `master`? This is unrelated and should not be committed.

Reply:
@justinhough, can you uncommit this?

Reply:
@justinhough, can you uncommitted this?

Reply:
@julio8a Just removed it in the latest commit.